### DISCUSSION TITLE
Can't use | into wezterm

### DISCUSSION THREAD

**Discussion by xmailla** [wezterm/wezterm#6193] [Q&A] [answered]
### What Operating System(s) are you running on?

macOS

### WezTerm version

wezterm 20240203-110809-5046fc22

### Ask your question!

Hello,

new with wezterm which I want to use to replace my old configuration based on Alacritty/iTerm2 and tmux. I did not make much configuration yet. There is a weird thing here, I can't use the "pipe" symbol in wezterm. Any help to debug appreciated.

Thank you.

**Comment by bew** [comment]
What keyboard layout do you use? And what keys are you supposed to press for the pipe symbol?

You can activate `debug_key_events` in your config to showcase what the terminal detects. 
Ref: https://wezfurlong.org/wezterm/config/lua/config/debug_key_events.html

**Reply by xmailla** [reply]
In fact, I finally found out that only SHIFT+ Right OPTION key is working not the left OPTION.
Can I fix that ?

**Comment by neogeographica** [comment, marked as answer]
Try setting send_composed_key_when_left_alt_is_pressed to true in the config: https://wezfurlong.org/wezterm/config/keyboard-concepts.html#macos-left-and-right-option-key

**Reply by xmailla** [reply]
Did that and it fixed my problem:
```
config.send_composed_key_when_left_alt_is_pressed = true
config.send_composed_key_when_right_alt_is_pressed = false
```

**Comment by oivindoh** [comment]
For anyone stumbling on this with a norwegian or similar utterly broken mac keyboard layout, you might want to also set `config.use_dead_keys = false` to regain the normal functionality of `~` after setting `config.send_composed_key_when_left_alt_is_pressed = true`
